esmvaltool/recipes/recipe_climwip_brunner2019_med.yml

```diff
@@ -0,0 +1,295 @@
+  - {dataset: IPSL-CM5A-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r3i1p1}
+  - {dataset: IPSL-CM5A-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r4i1p1}
+
+  - {dataset: IPSL-CM5A-MR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: IPSL-CM5B-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: MIROC-ESM-CHEM, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: MIROC-ESM, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  # MIROC5 has extended historical runs (overlap with RCP85) the pre-processor can't merge
+  # the first ensemble member for rsus (for the other ones it works)
+  # - {dataset: MIROC5, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: MIROC5, project: CMIP5, exp: [historical, rcp85], ensemble: r2i1p1}
+  - {dataset: MIROC5, project: CMIP5, exp: [historical, rcp85], ensemble: r3i1p1}
+
+  - {dataset: MPI-ESM-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: MPI-ESM-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r2i1p1}
+  - {dataset: MPI-ESM-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r3i1p1}
+
+  - {dataset: MPI-ESM-MR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: MRI-CGCM3, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: MRI-ESM1, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: NorESM1-ME, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: NorESM1-M, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+
+obs_data: &obs_data  # for climwip performance metrics
+  - {dataset: ERA5, project: native6, type: reanaly, version: '1', tier: 3}
+
+preprocessors:
+  climwip_general: &general
+    regrid:
+      target_grid: 2.5x2.5
+      scheme: linear
+    mask_landsea:
+      mask_out: sea
+
+  climatological_mean: &mean
+    climate_statistics:
+      operator: mean
+
+  climatological_std: &std
+    climate_statistics:
+      operator: std_dev
+
+  region: &region
+    <<: *general
+    extract_shape:
+      shapefile: 'shapefiles/SREX/MED/MED.shp'  # <-----
+      method: contains
+      crop: true
+
+  region_mean:
+    <<: *region
+    <<: *mean
+
+  region_std:
+    <<: *region
+    <<: *std
+
+  temperature_anomalies:
+    custom_order: true
+    <<: *region
+    area_statistics:
+      operator: mean
+    annual_statistics:
+      operator: mean
+    anomalies:
+      period: full
+      reference:
+        start_year: 1981
+        start_month: 1
+        start_day: 1
+        end_year: 2010
+        end_month: 12
+        end_day: 31
+      standardize: false
+
+diagnostics:
+  calculate_weights_climwip:
+    variables:
+      tas_CLIM: &common_settings
+        short_name: tas
+        start_year: 1995
+        end_year: 2014
+        mip: Amon
```

Comment:
You could consider to move the common settings definition all the way to the top, near the datasets. Then it's easier to see that all datasets share these time spans and mip (for CLIM you could overwrite it).